Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -43,7 +43,7 @@ static const float s_gammaLUT[] = {1.0f, 1.7f, 2.2f, 1.0f};
 
 void BPInit()
 {
-  memset(&bpmem, 0, sizeof(bpmem));
+  memset(static_cast<void*>(&bpmem), 0, sizeof(bpmem));
```

Comment:
This is, again, hiding an issue rather than taking the time to fix it properly.

Reply:
okay. I will fix it.